pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -533,6 +537,22 @@ func (m *manager) updateContainerCPUSet(ctx context.Context, containerID string,
 	// helpers_linux.go similar to what exists for pods.
 	// It would be better to pass the full container resources here instead of
 	// this patch-like partial resources.
+
+	if runtime.GOOS == "windows" && utilfeature.DefaultFeatureGate.Enabled(kubefeatures.WindowsCPUAndMemoryAffinity) {
```

Comment:
If we are on windows and the feature gate isn't enabled - shouldn't we just return nil / no-op here?
I don't think it makes sense to set LinuxContainerResources setting on Windows..

Reply:
agree it doesn't make sense, but it would be a change in behavior....

Reply:
Containerd doesn't at these extra fields on Windows right?
I think that wouldn't result in any actual behavior changes

Reply:
Will leave the decision up to others tho :)

Reply:
After getting this to build on Linux I've had to split this out to two different files, so I've changed this to not set the `LinuxContainerResources ` on Windows.